app/script/view_model/content/PreferencesDeviceDetailsViewModel.js

```diff
@@ -37,56 +37,32 @@ z.viewModel.content.PreferencesDeviceDetailsViewModel = class PreferencesDeviceD
     this.conversationRepository = repositories.conversation;
     this.cryptographyRepository = repositories.cryptography;
     this.logger = new z.util.Logger('z.viewModel.content.PreferencesDeviceDetailsViewModel', z.config.LOGGER.OPTIONS);
-    this.locationRepository = repositories.location;
```

Comment:
Is the locationRepo used somewhere else? Maybe we can totally remove it?

Reply:
It is still used.

Reply:
It should not be used anywhere. We can remove geolocation and CSP headers related to it.